packages/adapters/bitcoin/src/utils/BitcoinConnector.ts

```diff
@@ -30,7 +30,7 @@ export namespace BitcoinConnector {
     /**
      * The purpose of the address
      */
-    purpose: 'payment' | 'ordinal' | 'stx'
+    purpose: 'payment' | 'ordinal' | 'stx' | 'ordinals' | 'stacks'
```

Comment:
is `stx` standardized anywhere else? given we were not really integrating stacks before, should we remove ?

Reply:
It might come from WalletKit, idk, asked in the Slack, will ping Gancho again